bashellite.sh

```diff
@@ -372,25 +348,25 @@ Validate_repo_metadata() {
   done
 
   # Parsing populated *_url.conf file and setting variable values accordingly.
-  if [[ -s "${script_dir}/_metadata/${repo_name}/rsync_url.conf" ]]; then
-    rsync_url="$(cat ${script_dir}/_metadata/${repo_name}/rsync_url.conf)";
-  elif [[ -s "${script_dir}/_metadata/${repo_name}/http_url.conf" ]]; then
-    http_url="$(cat ${script_dir}/_metadata/${repo_name}/http_url.conf)";
+  if [[ -s "${metadata_tld}/${repo_name}/rsync_url.conf" ]]; then
+    rsync_url="$(cat ${metadata_tld}/${repo_name}/rsync_url.conf)";
+  elif [[ -s "${metadata_tld}/${repo_name}/http_url.conf" ]]; then
+    http_url="$(cat ${metadata_tld}/${repo_name}/http_url.conf)";
     # Ensures consistency by dropping trailing "/" from $http_url.
     http_url="${http_url%\/}";
-  elif [[ -s "${script_dir}/_metadata/${repo_name}/pypi_url.conf" ]]; then
-    pypi_url="$(cat ${script_dir}/_metadata/${repo_name}/pypi_url.conf)";
-  elif [[ -s "${script_dir}/_metadata/${repo_name}/aptmirror_url.conf" ]]; then
-    aptmirror_url="$(cat ${script_dir}/_metadata/${repo_name}/aptmirror_url.conf)";
+  elif [[ -s "${metadata_tld}/${repo_name}/pypi_url.conf" ]]; then
+    pypi_url="$(cat ${metadata_tld}/${repo_name}/pypi_url.conf)";
+  elif [[ -s "${metadata_tld}/${repo_name}/aptmirror_url.conf" ]]; then
+    aptmirror_url="$(cat ${metadata_tld}/${repo_name}/aptmirror_url.conf)";
   else
     Warn "Please place the mirror URL of the repo in ONE of the following only:";
-    Warn " rsync => ${script_dir}/_metadata/${repo_name}/rsync_url.conf";
-    Warn " http => ${script_dir}/_metadata/${repo_name}/http_url.conf";
-    Warn " pypi => ${script_dir}/_metadata/${repo_name}/pypi_url.conf";
-    Warn " aptmirror => ${script_dir}/_metadata/${repo_name}/aptmirror_url.conf";
+    Warn " rsync => ${metadata_tld}/${repo_name}/rsync_url.conf";
+    Warn " http => ${metadata_tld}/${repo_name}/http_url.conf";
+    Warn " pypi => ${metadata_tld}/${repo_name}/pypi_url.conf";
+    Warn " aptmirror => ${metadata_tld}/${repo_name}/aptmirror_url.conf";
     Warn "Please place any desired filter parameters, (aka includes and/or excludes), in:";
-    Warn " => ${script_dir}/_metadata/${repo_name}/repo_filter.conf";
-    Fail "Place the URL(s) of the repo (${repo_name}) in one, and ONLY one, \"_metadata/*_url.conf\" file and try again.";
+    Warn " => ${metadata_tld}/${repo_name}/repo_filter.conf";
+    Fail "Place the URL(s) of the repo (${repo_name}) in one, and ONLY one, \"${script_name}/${repo_name}/<provider>_url.conf\" file and try again.";
```

Comment:
I don't see where script_name is being defined.

Reply:
I stripped this var out, and must have missed this reference. I will fix and submit another commit.